content/docs/introducing-jsx.md

```diff
@@ -6,17 +6,17 @@ prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Considera la declaración de esta variable:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Esta divertida sintaxis de etiquetas no es ni un string ni HTML.
 
-It is called JSX, and it is a syntax extension to JavaScript. We recommend using it with React to describe what the UI should look like. JSX may remind you of a template language, but it comes with the full power of JavaScript.
+Esto es llamado JSX, y es una extensión de la sintaxis de JavaScript. Recomendamos usarlo con React para describir cómo debería ser la interfaz de usuario. JSX puede recordarte a un lenguaje de plantillas, pero viene con todo el poder de JavaScript.
 
-JSX produces React "elements". We will explore rendering them to the DOM in the [next section](/docs/rendering-elements.html). Below, you can find the basics of JSX necessary to get you started.
+JSX produce "elementos" de React. Exploraremos como renderizarlos en el DOM en la [siguiente sección](/docs/rendering-elements.html). A continuación puedes encontrar lo básico de JSX que será necesario para empezar.
```

Comment:
> Se llama JSX,

is more an adequate translation than

> Esto es llamado JSX

My reasoning being that if it was really saying "esto se llama" then it would say "this is called", but the original text refers to JSX as tough it has been introduced before using "it".

On the other hand, I agree with @montogeek's suggestion altough I'd remove `que será` leaving us with

> A continuación puedes encontrar lo básico de JSX necesario para empezar.

What are your thoughts?

Reply:
Idk but I think that `Es llamado` sounds better, and yes, I'm gonna change the other comment to `que necesitas para empezar`.

Reply:
> > Se llama JSX, y es una extensión de la sintaxis de JavaScript.
Okay, I'm gonna change it, thanks Guillem!